frontend/src/App.tsx

```diff
@@ -213,84 +233,63 @@ services:
-                    </p>
-
-                    <p className="mb-2 code-block">
-                      <CodeBlock
-                        text={servicesText}
-                        language='yaml'
-                        showLineNumbers={true}
-                        theme={dracula}
-                        startingLineNumber={72}
-                      />
-                    </p>
-
-                    <p className="mb-2 mt-4">
-                      Once you have done so, commit those changes:
-                    </p>
-                    <p className="mb-2 mt-4">
-                      <CopyButton className="hidden sm:inline-block w" copyText="git commit -am 'Add Redis service and relationship.'">
-                        <code className="px-4">git commit -am 'Add Redis service and relationship.'</code>
-                      </CopyButton>
-                    </p>
-                    <p className="mb-2 mt-4">
-                      And then push to Upsun:
-                    </p>
-                    <p className="mb-2 mt-4">
-                      <CopyButton className="hidden sm:inline-block w" copyText="upsun push">
-                        <code className="px-4">upsun push</code>
-                      </CopyButton>
-                    </p>
-                    <blockquote className="mb-2 mt-4 p-4 rounded-md border border-upsun-violet-600 bg-upsun-violet-900">
-                      <p className="mb-2 mt-2">
-                        <strong>Heads up!</strong>
-                      </p>
-                      <p className="mb-2 mt-2">
-                        Once you push this service, Upsun will build and <em>attempt</em> to deploy your project.
-                        <em>However</em>, until you define the resources you would like to make available for that service,
-                        the deployment will fail. Run the next command to define resources and move onto the next step.
-                      </p>
-                    </blockquote>
-                    <p className="mb-2 mt-4">
-                      Once Upsun asks for the resources definition for Redis, run:
-                    </p>
-                    <p className="mb-2 mt-4">
-                      <CopyButton className="hidden sm:inline-block w" copyText="upsun resources:set --size redis_persistent:0.5 --disk redis_persistent:512">
-                        <code className="px-4">upsun resources:set --size redis_persistent:0.5 --disk redis_persistent:512</code>
-                      </CopyButton>
-                    </p>
-                    <p className="mb-2 mt-4">
-                      When the activity has completed, just refresh this page.
-                    </p>
+                      We'll use this preview environment as a sandbox to stage the addition of a Redis service. Once happy, we'll bring it into production using <code className='px-1'>git merge</code>.
```

Comment:
```suggestion
                      We'll use this preview environment as a sandbox to stage the addition of a Redis service. Once happy, we'll bring it into production using <code className='px-1'>upsun merge</code>.
```

Reply:
Good point. 

The idea was to reinforce our Git-driven product. How do you feel about the following?

```suggestion
                      We'll use this preview environment as a sandbox to stage the addition of a Redis service. Once happy, we'll  commit the changes using <code className='px-1'>git</code> and then merge it into production.
```